packages/frontend/pages/_document.tsx

```diff
@@ -0,0 +1,19 @@
+import { Head, Html, Main, NextScript } from 'next/document'
+
+const Document = () => {
+  return (
+    <Html>
+      <link rel="icon" href="/favicon.ico" sizes="any" />
```

Comment:
I've never done this in `_document.tsx` (always in `_app.tsx` inside a `<Head>` wrapper), so this question might be dumb: But why is this placed not inside `<Head>…</Head>` but instead before a self-closing `<Head />`?

Reply:
As discussed, I must have totally spaced out here and didn't realize I wasn't placing them as children to the `Head` component. Thanks for spotting this!